central/globaldb/postgres.go

```diff
@@ -75,13 +68,27 @@ var (
 // GetPostgres returns a global database instance
 func GetPostgres() *pgxpool.Pool {
 	pgSync.Do(func() {
-		_, config, err := GetPostgresConfig()
+		sourceMap, dbConfig, err := pgconfig.GetPostgresConfig()
 		if err != nil {
 			log.Fatalf("Could not parse postgres config: %v", err)
 		}
 
+		// Get the active database name for the connection
+		activeDB := pgconfig.GetActiveDB()
+
+		// Create the active database if necessary
+		if !pgadmin.CheckIfDBExists(dbConfig, activeDB) {
+			err = pgadmin.CreateDB(sourceMap, dbConfig, pgadmin.AdminDB, activeDB)
```

Comment:
We do not need to try to create the database if it does not exists. We let the migrator ensure that the current active database presents.